### PR TITLE
Add some more options for nginx

### PR DESCRIPTION
**Pull request by isqad**
# What's inside

* Added `-c` option for passing ssl certificates
* Allow to pass more than one port

# Checklist:

- [x] I have added tests
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by isqad** [comment]
:up:

File: lib/dip/cli/nginx.rb [hunk 1]
```diff
@@ -17,11 +17,13 @@ class Nginx < Thor
       method_option :net, aliases: '-t', type: :string, default: "frontend",
                           desc: 'Container network name'
       method_option :publish, aliases: '-p', type: :string, default: "80:80",
-                              desc: 'Container port'
+                              desc: 'Container port(s). For more than 1 ports separate by comma'
       method_option :image, aliases: '-i', type: :string, default: "bibendi/nginx-proxy:latest",
                             desc: 'Docker image name'
       method_option :domain, aliases: '-d', type: :string, default: "docker",
                              desc: 'Top level domain'
+      method_option :certs, aliases: '-c', type: :string, default: ".",
```
Comment (bibendi) [inline lib/dip/cli/nginx.rb]: Let's remove a default value

File: README.md [hunk 2]
```diff
@@ -298,6 +299,20 @@ curl www.bar-app.docker/api/v1/quz
 curl www.bar-app.docker/api/v1/baz_service/qzz
 ```
 
+#### Pass SSL certificates
+
+```sh
+dip nginx up -c /home/$(whoami)/ssl_certificates
```
Comment (bibendi) [inline README.md]:
```suggestion
dip nginx up -c $HOME/ssl_certificates
```

File: lib/dip/cli/nginx.rb [hunk 3]
```diff
@@ -17,11 +17,13 @@ class Nginx < Thor
       method_option :net, aliases: '-t', type: :string, default: "frontend",
                           desc: 'Container network name'
       method_option :publish, aliases: '-p', type: :string, default: "80:80",
```
Comment (bibendi) [inline lib/dip/cli/nginx.rb]: Let's use an array type.

https://github.com/erikhuda/thor/wiki/Method-Options

**Comment by isqad** [comment]
fixed

File: README.md [hunk 4]
```diff
@@ -298,6 +298,20 @@ curl www.bar-app.docker/api/v1/quz
...
 
+#### Pass SSL certificates
+
+```sh
+dip nginx up -c $HOME/ssl_certificates
+```
+
+#### Publish more than one port to localhost
+
+Just pass as list splitted by comma:
```
Comment (bibendi) [inline README.md]:
```suggestion
Just pass a list, separated by a space:
```

**Comment by bibendi** [comment]
One more small fix and we are ready to :ship:

**Comment by bibendi** [comment]
Started rolling out the release https://travis-ci.org/bibendi/dip/builds/608039149
It will have compiled in one hour.

**Pull request merged**